bundles/org.openhab.binding.dirigera/src/main/resources/OH-INF/thing/temperature-light.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dirigera"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="temperature-light">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+
+		<label>Temperature Light</label>
+		<description>Light with color temperature support</description>
+
+		<channels>
+			<channel id="power-state" typeId="switch">
```

Comment:
Power (system.power) has a system channel, prefer that you use that one:
https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types
Also applies to other `Things`

Reply:
Done